packages/redux/src/recentlyViewed/__tests__/reducer.test.ts

```diff
@@ -2,21 +2,22 @@ import * as fromReducer from '../reducer';
 import {
   expectedLocalPayload,
   expectedRemotePayload,
-} from 'tests/__fixtures__/recentlyViewed/getRecentlyViewed';
+} from 'tests/__fixtures__/recentlyViewed';
 import reducer, { actionTypes } from '..';
 
 let initialState;
+const randomAction = { type: 'this_is_a_random_action' };
 
 describe('Recently Viewed reducer', () => {
   beforeEach(() => {
-    initialState = reducer();
+    initialState = fromReducer.INITIAL_STATE;
```

Comment:
Question: Why was this change necessary?

Reply:
As I checked on other files it's used two approaches for this scenario. I found that it's being frequently changing the initialState to reducer's Initial state const, or in a few cases, uses a reducer with InitialState as a parameter. But the first approach is mostly used instead of the second one.

Reply:
I understand but I think that the other way was better, because calling the reducer with an empty state should return the initial state by default. Imagine if tomorrow we change the reducer logic to return something else that is not exactly from the `INITIAL_STATE` variable. 

However, you can let it this way, as it is not critical.